src/components/pdf-viewer.vue

```diff
@@ -1130,8 +1142,16 @@ export default class PdfViewer extends Vue {
     this.$emit("after-created", pdfApp.PDFViewerApplication);
   }
 
-  private mounted() {
+  private async mounted() {
     this.addPrintContainer();
+    if (!viewerElementReady()) {
+        // If it hasn't been mounted we'll wait for up to 5 seconds to see if it
+        // gets mounted in the DOM; if it still isn't mounted we'll give up
+        const foundAfterWaiting = await waitForAppConfig();
+        if (!foundAfterWaiting) {
+            console.warn("Could not find app container; PDF viewer will probably not work");
+        }
+    }
```

Comment:
Sorry I meant to come back and update the PR to explain these changes. I'm using vue-pdf-app inside a vuetify dialog and so the element isn't always in the page when it tries to start up, which causes it to fail; what this does is it checks to see if it can find the element in the page and if not it'll wait for up to 5 seconds (non-blocking) to see if it shows up.

This fixed the issue I was having with loading it in the page about 90% of the time; there is still something somewhere which can keep it from loading, but most of the time it works for me now.

Reply:
did that make sense? anything else you'd like me to do with this?